.travis.yml

```diff
@@ -32,3 +37,8 @@ matrix:
 
     - python: "3.6"
       env: TOXENV=py36-flake8
```

Comment:
py36-flake8 and py27-flake8 can be removed now, i.e.:

```yml
    - python: "2.7"
      env: TOXENV=py27-flake8

    - python: "3.6"
      env: TOXENV=py36-flake8
```

Reply:
Hi @sevein did you notice when you run tox with that newer setting then Flake8 doesn't run locally? That's often beneficial to the developer before pushing. I'm not sure though, maybe it's doing something different, or my configuration needs changing?

Reply:
This needs to go!